dev-util/packer/packer-1.8.5.ebuild

```
@@ -0,0 +1,35 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Always aim to use the latest available EAPI, 8 should work here
```suggestion
EAPI=8
```